packages/office-ui-fabric-react/src/components/ResizeGroup/ResizeGroup.base.tsx

```diff
@@ -186,6 +187,46 @@ export const getNextResizeGroupStateProvider = (measurementCache = getMeasuremen
     };
   }
 
+  /**
+   * Scale the given data based on the measurements already in cache and based on the current container dimension
+   * This is useful to optimize changes to ResizeGroup props which do not affect scaling. In that case,
+   * we could avoid measuring the resize group.
+   * @param data - The initial data point to start measuring.
+   * @param onReduceData - Function that transforms the data into something that should render with less width/height.
+   * @param onGrowData - Function that transforms the data into something that may take up more space when rendering.
+   */
+  function scaleDataBasedOnMeasurementsInCache(data: any, onReduceData: (prevData: any) => any, onGrowData?: (prevData: any) => any): any {
```

Comment:
Is there a test we can add to cover this new behavior?

Reply:
Yes.....adding that....

Reply:
Done.